src/scss/components/_testimony.scss

```diff
@@ -0,0 +1,56 @@
+/**
+  * @parent gaiden_css.components
+  * @stylesheet gaiden_css.components.testimony.css Testimony
+  *
+  * ## Services
+  * @demo demo/gaiden-css/components/testimony/demo.html
+**/
+.wrap__testimony__itens {
+  float: left;
+}
+
+.testimony {
```

Comment:
@MarcoBrunoBR can we use 'dialog' name? `testimony` appears to be a specific element. `dialog` or `ballon`.